cms/static/js/certificates/views/signatory_details.js

```diff
@@ -73,6 +73,7 @@ function ($, _, str, Backbone, gettext, TemplateUtils, ViewUtils, BaseView, Sign
                     actionableModel.save({}, {
                         success: function() {
                             actionableModel.setOriginalAttributes();
+                            self.model.setOriginalAttributes();
```

Comment:
why we need to set original attributes of signatory model in case of successful save?